spec/scrip-spec.js

```diff
@@ -0,0 +1,45 @@
+/** Testing file using Jasmine on emailParse.js. */
+let app= require("../project/src/main/webapp/emailParse.js");
+let MLClass = app.MLModelEmailParse;
+
+const modelClass = new MLClass();
+describe("getScoreToEmail",function(){
+    it("The function only returns map with ML answer and confidence score with email body for later reference",function() {
```

Comment:
The description of the test will be shown right after the description of the `describe` and should read like a proper sentence. So this one would be `getScoreToEmail only returns map...` and you can remove `The function` from the test description.